wrapanapi/systems/ec2.py

```diff
@@ -353,7 +353,66 @@ def deploy(self, *args, **kwargs):
         return self.system.create_vm(image_id=self.uuid, *args, **kwargs)
 
 
-class EC2System(System, VmMixin, TemplateMixin, StackMixin):
+class EC2Vpc(Network, _TagMixin):
+    def __init__(self, system, raw=None, **kwargs):
+        """
+        Constructor for an EC2Network tied to a specific system.
+
+        Args:
+            system: an EC2System object
+            raw: the boto.ec2.network.Network object if already obtained, or None
+            uuid: unique ID of the network
+        """
+        self._uuid = raw.id if raw else kwargs.get('uuid')
+        if not self._uuid:
+            raise ValueError("missing required kwarg: 'uuid'")
+
+        super(EC2Vpc, self).__init__(system, raw, **kwargs)
+
+        self._api = self.system.ec2_connection
+
+    @property
+    def _identifying_attrs(self):
+        return {'uuid': self._uuid}
+
+    @property
+    def name(self):
```

Comment:
Oops. Name is name if name else uuid. 
I had problem in refresh because I had get_network there without setting argument as id.
It should be working correctly now.